app/assets/javascripts/editor.js

```diff
@@ -1,13 +1,16 @@
 $E = {
   initialize: function(args) {
     args = args || {}
+    // title
+    $E.title = $('#title')
+    // textarea
     args['textarea'] = args['textarea'] || 'text-input'
     $E.textarea = $('#'+args['textarea'])
-    $E.title = $('#title')
+    $E.textarea.bind('input propertychange',$E.save)
+    // preview
     args['preview'] = args['preview'] || 'preview'
```

Comment:
No additions here, just rearranged stuff for greater readability. Sometime soon, I'd like to rewrite this part to use JavaScript ES6's default parameters: `initialize(textarea = 'text-input')`